sdk/desktopvirtualization/arm-desktopvirtualization/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@azure/arm-desktopvirtualization",
+  "sdk-type": "mgmt",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for DesktopVirtualizationAPIClient.",
+  "version": "1.0.0-beta.1",
+  "engines": { "node": ">=12.0.0" },
+  "dependencies": {
+    "@azure/core-paging": "^1.2.0",
+    "@azure/core-client": "^1.0.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.1.0",
+    "tslib": "^2.2.0"
+  },
+  "keywords": ["node", "azure", "typescript", "browser", "isomorphic"],
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/arm-desktopvirtualization.d.ts",
+  "devDependencies": {
+    "@microsoft/api-extractor": "^7.18.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-multi-entry": "^3.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "mkdirp": "^1.0.4",
+    "rollup": "^1.16.3",
+    "rollup-plugin-sourcemaps": "^0.4.2",
+    "typescript": "~4.2.0",
+    "uglify-js": "^3.4.9",
+    "@azure/identity": "2.0.0-beta.6",
```

Comment:
@sarangan12 Could you help review this PR  https://github.com/Azure/autorest.typescript/pull/1233 ASAP so that cases like this won't happen again. Thanks